pkg/controller/cincinnati/new.go

```diff
@@ -62,6 +63,31 @@ data_directory = "/tmp/cincinnati/graph-data"
 [[plugin_settings]]
 name = "edge-add-remove"`
 
+// TOML when graph data source is an init container
```

Comment:
Did you consider using just one template, and putting a conditional around the github part? Perhaps you could have that rendered only if `instance.GraphDataImage != ""`?

Reply:
I am in agreement with this idea: https://github.com/openshift/cincinnati-operator/pull/10#discussion_r408837314